src/controllers/reviewController.js

```diff
@@ -0,0 +1,32 @@
+const reviewService = require('../services/reviewService');
+
+const postReview = async (req, res) => {
+  const { productId, title, content, image_url } = req.body;
```

Comment:
```suggestion
  const { productId, title, content, image_url } = req.body;
  
  if ( !productId || !title || !content || !imageUrl ) { ... }
```
body에 의도한 값이 들어오지 않았을 때의 예외 처리를 해주세요.

Reply:
모든 네이밍 컨벤션이 동일해야 합니다. (CamelCase, snake_case를 섞어서 사용하면 안됨)